src/shared/components/oncoprint/TooltipUtils.ts

```diff
@@ -39,6 +39,20 @@ function makeGenePanelPopupLink(gene_panel_id:string) {
     });
     return anchor;
 }
+
+export function linebreakGenesetId(
+    genesetId: string
+): string {
+    return (
+        // encode the string as the textual contents of an HTML element
+        $('<div>').text(genesetId).html()
+        // Include zero-width spaces to allow line breaks after punctuation in
```

Comment:
nice idea, and helpful comment

Reply:
I considered building it in OncoprintJS itself, but it sort of depends on knowledge about what the headers will generally look like and where they can be sensibly split, so I had to keep it specific to gene set IDs.